xbmc/cores/VideoPlayer/VideoRenderers/OverlayRendererGL.cpp

```diff
@@ -207,15 +207,17 @@ COverlayTextureGL::COverlayTextureGL(CDVDOverlayImage* o)
     float center_x = (0.5f * o->width  + o->x) / o->source_width;
     float center_y = (0.5f * o->height + o->y) / o->source_height;
 
-    m_width  = (float)o->width  / o->source_width;
-    m_height = (float)o->height / o->source_height;
+    m_width = static_cast<float>(o->width);
+    m_height = static_cast<float>(o->height);
     m_pos    = POSITION_RELATIVE;
 
     {
       /* render aligned to screen to avoid cropping problems */
-      m_align  = ALIGN_SCREEN;
+      m_align = ALIGN_SCREEN_AR;
```

Comment:
Comment above is now different because the meaning of AR is different than the old type

Reply:
i can remove the comment? I don't find it very useful